src/components/editor/AddReferences.vue

```diff
@@ -42,7 +42,7 @@
               class="search-button"
               size="sm"
               text="Button"
-              :disabled="!doiValid || createReferenceFlags.loading"
+              :disabled="doiValid === false || doiValid === null || createReferenceFlags.loading"
```

Comment:
I think this could be shortened using jscript falsy evaluation

`if !doiValue || createReferencesFlags.loading`